include/contractor/graph_contractor_adaptors.hpp

```diff
@@ -20,6 +20,9 @@ std::vector<ContractorEdge> adaptToContractorInput(InputEdgeContainer input_edge
 
     for (const auto &input_edge : input_edge_list)
     {
+        if (input_edge.data.weight == INVALID_EDGE_WEIGHT)
+            continue;
```

Comment:
No previously we filtered these edges out the edge loading routine. We we loading one struct at a time, so we could just skip them.